tekton/pipeline_run_test.go

```diff
@@ -20,11 +20,13 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
-	"k8s.io/apimachinery/pkg/types"
 	"os"
 	"reflect"
 	"strings"
 
+	"k8s.io/apimachinery/pkg/runtime"
```

Comment:
Remove all the changes in this file. No test is needed as you will be adding no new functionality.

Reply:
ok, done, will be part of the second PR (or, in this case, it _will not be_ part of it ;) ).